src/helpers.js

```diff
@@ -3,8 +3,10 @@
 import ChildProcess from 'child_process'
 import { createFromProcess } from 'process-communication'
 import { join } from 'path'
-// eslint-disable-next-line import/no-extraneous-dependencies
+
+/* eslint-disable import/no-extraneous-dependencies, import/extensions */
 import { Disposable } from 'atom'
+/* eslint-enable import/no-extraneous-dependencies, import/extensions */
```

Comment:
I was explicitly disabling the two rules here on the off chance that there is another issue with the line besides those two. If you want I can just disable all checking for the line though.